extensions/blocks/markdown/index.js

```diff
@@ -67,4 +67,14 @@ export const settings = {
 	edit,
 
 	save,
+
+	example: {
+		attributes: {
+			source:
+				'## ## Try Markdown\n\n' +
```

Comment:
Why the extra `##`?

Reply:
Just to give a visual hint it's markdown.

Reply:
I like it, and it makes me wonder if we should do the same with the bold and italic too, to show even more Markdown syntax in the preview?

Reply:
I tried, but couldn't do the same with those. They just got converted into the tags.